python/ray/tune/hyperband.py

```diff
@@ -32,24 +32,29 @@ class HyperBandScheduler(FIFOScheduler):
     and band and will spill over to new brackets/bands accordingly.
     """
 
-    def __init__(self, max_iter=200, eta=3):
+    def __init__(self, max_iter=81, eta=None):
         """
         args:
-            max_iter (int): maximum iterations per configuration
-            eta (int): # defines downsampling rate (default=3)
+            max_iter (int): maximum iterations per configuration. Altering
```

Comment:
Can we change this to allow specifying different time attributes, similar to MedianStoppingRule?